kokoro/oak_containers_presubmit.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+set -o pipefail
+
+export CI=kokoro
+
+export RUST_BACKTRACE=1
+export RUST_LOG=debug
+export XDG_RUNTIME_DIR=/var/run
+
+./scripts/docker_pull
+./scripts/docker_run nix develop --command just oak_containers_kokoro
```

Comment:
It may be enough to only load the .#containers shell (which is smaller than the default one)

You can try locally, but make sure you test by calling `nix develop --ignore-environment --keep HOME --keep USER .#containers` to make sure no stuff from the host leaks in.

Reply:
On that note, I should switch the regular `presubmit.sh` back to `.#ci` as well as we no longer need containers there.